.github/workflows/common-java-gradle-release.yml

```diff
@@ -0,0 +1,108 @@
+# This is a basic release workflow for Octopus gradle Java modules with publishing to Sonatype Nexus and/or image registry
+name: Gradle Release
+
+on:
+  workflow_call:
+    inputs:
+      java-version:
+        required: false
+        type: string
+        default: '8'
+      docker-image:
+        description: 'Docker image name'
+        required: false
+        type: string
+        default: ''
+      increment-version-level:
+        description: 'One of: {major, premajor, minor, preminor, patch, prepatch, prerelease}'
+        required: false
+        type: string
+        default: patch
+    secrets:
+      OSSRH_USERNAME:
+        required: true
+      OSSRH_TOKEN:
+        required: true
+      GPG_PASSPHRASE:
+        required: true
+      GPG_PRIVATE_KEY:
+        required: true
+
+jobs:
+  prepare-build-publish-release:
+    runs-on: ubuntu-latest
+    environment: Prod
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Get latest tag
+        id: tag_version
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: ${{ github.repository }}
+          regex: '^v([0-9]+)\..*'
+
+      - name: Prepare new tag version
+        uses: actions-ecosystem/action-bump-semver@v1
+        id: new-tag-version
+        with:
+          current_version: ${{ steps.tag_version.outputs.tag }}
+          level: ${{ inputs.increment-version-level }}
+
+      - name: Get new version
+        uses: octopusden/octopus-base/.github/actions/get-version@v1.0
+        id: new-version
+        with:
+          tag-value: ${{ steps.new-tag-version.outputs.new_version }}
+          exit-if-empty: true
+
+      - run: |
```

Comment:
Just a note: unnamed step seems not a good practice.

Reply:
fixed